appium-tests/android/android.spec.js

```diff
@@ -21,10 +21,13 @@
  *
 */
 
-// these tests are meant to be executed by Cordova Medic Appium runner
-// you can find it here: https://github.com/apache/cordova-medic/
+// these tests are meant to be executed by Cordova ParaMedic Appium runner
+// you can find it here: https://github.com/apache/cordova-paramedic/
 // it is not necessary to do a full CI setup to run these tests
-// just run "node cordova-medic/medic/medic.js appium --platform android --plugins cordova-plugin-camera"
+// for android 5.1, run:
+//      node cordova-paramedic/main.js --platform android --plugin cordova-plugin-camera --skipMainTests --saucePlatformVersion "5.1" --target api22
```

Comment:
So, while that may be how paramedic right now was designed, that's actually counter-productive when working with appium. Appium will actually select the correct emulator (and even start it for you) based on the `platformName`, `platformVersion` and `deviceName` capabilities you provide. These capabilities apply regardless if you are running the test locally or on Sauce Labs.

I think, moving forward, for a paramedic refactor I am gathering use cases for / noting issues to address, we would want to expose the appium/selenium capabilities more completely. This would also reduce the difference between local test executions and executions on Sauce Labs, by allowing Appium to worry about managing the emulator, rather than managing it in paramedic.

I will change the instructions back, but just a heads up that I think we should change this behaviour in the future.